clippy_lints/src/utils/internal_lints.rs

```diff
@@ -207,3 +218,31 @@ impl<'a, 'tcx: 'a> Visitor<'tcx> for LintCollector<'a, 'tcx> {
         NestedVisitorMap::All(&self.cx.tcx.hir)
     }
 }
+
+
+
+pub struct PreferFxHash;
+
+impl LintPass for PreferFxHash {
+    fn get_lints(&self) -> LintArray {
+        lint_array!(PREFER_FX_HASH)
+    }
+}
+
+impl EarlyLintPass for PreferFxHash {
+    fn check_ident(&mut self, cx: &EarlyContext, ident: Ident) {
```

Comment:
This seems to be the first EarlyLintPass Herr, so you want to `use syntax::ast::*;`.